src/brogue/MainMenu.c

```diff
@@ -842,7 +842,9 @@ void mainBrogueJunction() {
                             rogue.playbackPaused = false;
                             pausePlayback();
                         }
-
+						if(rogue.gameHasEnded)
```

Comment:
A few general comments on style.

* Space after if before open parenthesis is preferred
* 4 space indents
* You might not need to add the define commented out in Rogue.h since you have it in the Makefile